modules/base/src/properties/imageinformationproperty.cpp

```diff
@@ -42,15 +42,25 @@ ImageInformationProperty::ImageInformationProperty(std::string_view identifier,
                                                    std::string_view displayName,
                                                    InvalidationLevel invalidationLevel,
                                                    PropertySemantics semantics)
-    : CompositeProperty(identifier, displayName, invalidationLevel, semantics)
-    , dimensions_("dimensions", "Dimensions", size2_t(0), size2_t(0),
-                  size2_t(std::numeric_limits<size_t>::max()), size2_t(1), InvalidationLevel::Valid,
-                  PropertySemantics("Text"))
+    : CompositeProperty(
+          identifier, displayName,
+          "A CompositeProperty holding properties to show a information about an image"_help,
+          invalidationLevel, semantics)
+    , dimensions_("dimensions", "Dimensions", "Image dimensions"_help, size2_t(0),
+                  {size2_t(0), ConstraintBehavior::Immutable},
+                  {size2_t(std::numeric_limits<size_t>::max()), ConstraintBehavior::Immutable},
+                  size2_t(1), InvalidationLevel::Valid, PropertySemantics("Text"))
+    , aspectRatio_("aspectRatio", "Aspect Ratio",
+                   OrdinalPropertyState<double>{0.0, 0.0, ConstraintBehavior::Immutable,
```

Comment:
there is no point in using the OrdinalPropertyState unless you plan to use on of the util::ordinalColor etc functions. here you might just as well just pass in all the args directly...